modules/ocf_irc/manifests/nodejs/apt.pp

```diff
@@ -0,0 +1,18 @@
+class ocf_irc::nodejs::apt {
+  package {
+    'apt-transport-https':;
+  }
+
+  # Install a newer version of Node.js for the Slack bridge and web IRC app
+  apt::key { 'nodejs':
+    id     => '9FD3B784BC1C6FC31A8A0A1C1655A0AB68576280',
+    source => 'https://deb.nodesource.com/gpgkey/nodesource.gpg.key';
+  }
```

Comment:
Since backporting nodejs from stretch looks to be a massive pain because of the number of build dependencies needed to actually build it, I think it would be better to go with the nodesource repo. Besides, I trust them enough to use it, and we can go with the 4.x repo instead of the 6.x repo if that helps at all with stability? When we upgrade to stretch then this won't be a problem any more, since we'll have 4.x nodejs by default, and the nodesource version of nodejs will only be used on flood until then, so it shouldn't really be much of a problem.